roles/k3s/tasks/k3s-worker.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Create k3s and kubelet directories
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    mode: '0755'
+    owner: root
+    group: root
+  loop:
+    - "{{ k3s_data_dir }}"
+    - /data/kubelet
```

Comment:
should probably be `"{{ kubelet_root_dir }}"` instead of being `/data/kubelet` hardcoded